TRIAGE_GUIDE.md

```diff
@@ -0,0 +1,33 @@
+
+# Overview
+
+SIG-Security holds weekly triage on [Mondays](https://lists.o3de.org/g/o3de-calendar/calendar). High priority issues can be triaged via Discord as and when required.
+
+The standard O3DE [triage guide](https://github.com/o3de/community/) should be used to cover process for accepting issues and setting standard labels.
```

Comment:
there does not seem to be a triage guide at this link?

Reply:
The intent is for community to own the common triage guide, so deferring to that. Have reached out privately to contributors who were adding this, also created general issue: https://github.com/o3de/community/issues/130